cpluginv1/specifier.go

```diff
@@ -37,4 +37,32 @@ type SpecifierParameter struct {
 	Default     string
 	Required    bool
```

Comment:
Can we please mark this field as deprecated?
```suggestion
	// Deprecated: Use ValidationTypeRequired instead.
	Required    bool
```